ext/mbstring/mbstring.c

```diff
@@ -1473,6 +1473,25 @@ static PHP_INI_MH(OnUpdate_mbstring_http_output_conv_mimetypes)
 	return SUCCESS;
 }
 /* }}} */
+
```

Comment:
If the PHP_INI_ENTRY() is commented out by preprocessor macros, then shouldn't the PHP_INI_MH() function be too?